tools/scripts/release/release-rollback.sh

```diff
@@ -5,11 +5,27 @@ set -ex
 log=$(mktemp)
 echo "Log directory is $log"
 
-exec 5>"$log"
-BASH_XTRACEFD="5"
+# Create a file descriptor that pipes stdout, stderr, and xtrace to a file and
+# only logs stdout and stderr to terminal.
+exec 3>"$log"
+exec > >(tee -a /dev/fd/3) 2>&1
 
-OWNER="build-trust"
-USER_TYPE="orgs"
+export BASH_XTRACEFD=3
+
+if [[ -z $OWNER ]]; then
+  OWNER="build-trust"
+fi
+
+USER_TYPE="users"
+
+user_type=$(gh api \
```

Comment:
small nitpick. For the same of readability. Instead of using `user_type` lowercase can you name it `type` as it matches the output you are looking at anyways? Since you are already using  uppercase `USER_TYPE` variable just above this line.

Reply:
Fixed, thanks